analyses/upstream-analysis/util/function-process-Seurat.R

```diff
@@ -0,0 +1,322 @@
+#############################################################################################
+#' Function to calculate cell cycle scores (S and G2M) for each cell.
+#' Genome specifies which gene set to use for cell cycle genes
+#' @param seurat_obj
+#' @param Genome
+#'
+#' @return
+#' @export
+#'
+#' @examples
+#' 
+Genome_Specific_Cell_Cycle_Scoring <- function(seurat_obj, Genome) {
+  
+  # Pull standard cell cycle genes for cell cycle scoring
+  human.s.genes <- cc.genes.updated.2019$s.genes
+  human.g2m.genes <- cc.genes.updated.2019$g2m.genes
+  
+  # Had to append "hg19-" to the beginning of all genes due to dual index reference genome used
+  dual.hg19.s.genes <- paste("hg19-", human.s.genes, sep = "")
+  dual.hg19.g2m.genes <- paste("hg19-", human.g2m.genes, sep = "")
+  
+  # Had to append "GRCh38-" to the beginning of all genes due to dual index reference genome used
+  dual.GRCh38.s.genes <- paste("GRCh38-", human.s.genes, sep = "")
+  dual.GRCh38.g2m.genes <- paste("GRCh38-", human.g2m.genes, sep = "")
+  
+  # Changed genes to title case for mice genes (Human genes are annotated in all CAPS)
+  mm10.s.genes <- str_to_title(cc.genes.updated.2019$s.genes)
+  mm10.g2m.genes <- str_to_title(cc.genes.updated.2019$g2m.genes)
+  
+  # Appended "GRCh38-" to the beginning of all genes due to dual index reference genome used
```

Comment:
Very minor, but just for clarity, maybe change this comment to correct that mm10-- is what's being appended

Reply:
```suggestion
  # Appended "mm10-" to the beginning of all genes due to dual index reference genome used
```

Reply:
Or if this script is updated to reflect the GRCm39 naming, than change mm10 to GRCm39

Reply:
This has been resolved here cb798b3318b1255b996c2f67365ef7bde923d827.